packages/ndla-ui/src/ContentTypeBlockQuote/ContentTypeBlockQuote.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { forwardRef } from "react";
+import { BlockQuote, BlockQuoteProps, BlockQuoteVariantProps } from "@ndla/primitives";
+import { ContentType } from "../ContentTypeBadge/ContentTypeBadgeNew";
+
+export type ContentTypeBlockQuoteVariant = "colored" | "neutral";
```

Comment:
Skal du ha støtte i ed for å sette begge disse, eller er neutral der kun for å ha en standardverdi?

Reply:
Dette er valgene i ED. Man kan velge at blockquote/framedcontent er nøytral (uten farge), eller så kan man velge at den får en farge som blir regnet ut fra innholdstypen.

Reply:
Men vil vi lagre neutral eller vil det tilsvare ingen verdi?

Reply:
Tenkte at både "neutral" og undefined kunne telle som neutral, så kan vi ha en inkrementell overgang.